### ISSUE TITLE
After installation in fedora 30 "conda init" does not work

### ISSUE THREAD

**Issue by onlymarcus** [conda/conda#8797]
I installed the anaconda as instructed on the site https://docs.anaconda.com/anaconda/install/linux/
After this, when you type conda init, the error message appears:

```
[root@localhost marcus]# conda init
WARNING: The conda.compat module is deprecated and will be removed in a future release.
no change     /root/anaconda3/condabin/conda
no change     /root/anaconda3/bin/conda
no change     /root/anaconda3/bin/conda-env
no change     /root/anaconda3/bin/activate
no change     /root/anaconda3/bin/deactivate
no change     /root/anaconda3/etc/profile.d/conda.sh
no change     /root/anaconda3/etc/fish/conf.d/conda.fish
no change     /root/anaconda3/shell/condabin/Conda.psm1
no change     /root/anaconda3/shell/condabin/conda-hook.ps1
no change     /root/anaconda3/lib/python3.7/site-packages/xonsh/conda.xsh
no change     /root/anaconda3/etc/profile.d/conda.csh
no change     /root/.bashrc
No action taken.

root@localhost marcus]# conda --version
conda 4.6.11
[root@localhost marcus]# conda info

     active environment : None
            shell level : 0
       user config file : /root/.condarc
 populated config files : /root/.condarc
          conda version : 4.6.11
    conda-build version : 3.17.8
         python version : 3.7.3.final.0
       base environment : /root/anaconda3  (writable)
           channel URLs : https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/free/linux-64
                          https://repo.anaconda.com/pkgs/free/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /root/anaconda3/pkgs
                          /root/.conda/pkgs
       envs directories : /root/anaconda3/envs
                          /root/.conda/envs
               platform : linux-64
             user-agent : conda/4.6.11 requests/2.21.0 CPython/3.7.3 Linux/5.1.5-300.fc30.x86_64 fedora/30 glibc/2.29
                UID:GID : 0:0
             netrc file : None
           offline mode : False

```

**Comment by soapy1** [comment]
The docs say: "conda init is available in conda versions 4.6.12 and later."

Please try to update conda, looks like you are on 4.6.11

**Comment by msarahan** [comment]
4.6.0 introduced conda init.  It may be that no change is happening because you already ran conda init as part of the installer process.
